packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js

```diff
@@ -5,7 +5,7 @@ import clsx from 'clsx';
 import ButtonBase from '../ButtonBase';
 import IconButton from '../IconButton';
 import withStyles from '../styles/withStyles';
-import ExpansionPanelContext from '../ExpansionPanel/Context';
+import ExpansionPanelContext from '../ExpansionPanel/ExpansionPanelContext';
```

Comment:
The title of the commit might suggest that you prefer `Context`, I do agree that it's unnecessarily long. I have no objection, as long as we change the other components 👌.

Reply:
Nah it's fine for most of the tooling. In vscode you get the display of the file and dirname so it is information duplication. But as far as I know many people still have an issue with `ExpansionPanel/index.js` because then they only see index.js in their IDE. As long as people still rely on the filename this is fine. But as always I recommend working these issues from both ends :)

Reply:
Alright, let's move to the next problem :).